release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -361,6 +361,14 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-auth"]
 === Authentication and authorization
 
+[id="ocp-4-15-auth-psa-telemetry"]
+==== Telemetry collection on clusters that violate pod security admission restricted enforcement
+
+With this release, additional Telemetry data is collected from clusters where pod security admission violations are occurring.
+This data collection will help Red Hat evaluate customer cluster readiness for pod security admission global restricted enforcement in the future.
```

Comment:
I didn't mention this, but we could mention that we only store the following:

- if a violation occurred in an openshift namespace
- if a violation occurred in a kubernetes system namespace
- if a violation occurred in a customer namespace.

We only track the [YES/NO] value for the three things. We don't collect any other (meta-)data.

Reply:
Thanks, updated!